netcam_rtsp.c

```diff
@@ -935,14 +945,17 @@ int netcam_setup_rtsp(netcam_context_ptr netcam, struct url_t *url){
 */
 int netcam_next_rtsp(unsigned char *image , netcam_context_ptr netcam){
```

Comment:
I really don't know :)

I think, given the function is specific to rtsp case (the one for the non-rtsp netcams works completely differently), keeping it in the rtsp specific file makes sense.

Removing the ifdefs makes sense. I've just pushed a commit that does that.